src/layouts/SiteLaunchPad/SiteLaunchPad.tsx

```diff
@@ -199,22 +179,22 @@ export const SiteLaunchPadPage = (): JSX.Element => {
   const {
     siteLaunchStatusProps,
     setSiteLaunchStatusProps,
+    pageNumber,
   } = useSiteLaunchContext()
   const { siteName } = useParams<{ siteName: string }>()
-  const [pageNumber, setPageNumber] = useState(
-    getInitialPageNumber(siteLaunchStatusProps)
-  )
 
   const errorToast = useErrorToast()
   const { data: role } = useGetCollaboratorRoleHook(siteName)
+
+  const isLoaded: boolean = !!siteName && !!role
   useEffect(() => {
-    if (!isSiteLaunchEnabled(siteName, role)) {
+    if (isLoaded && !isSiteLaunchEnabled(siteName, role)) {
       errorToast({
         id: "no_access_to_launchpad",
         description: "You do not have access to this page.",
       })
     }
-  }, [siteName, errorToast, role])
+  }, [siteName, errorToast, role, isLoaded, siteLaunchStatusProps])
```

Comment:
Is `siteLaunchStatusProps` a dependency for this?

Reply:
nope, good catch! 8484744fba0d59973275b3341d90f8e840221038

Reply:
nope, good catch! https://github.com/isomerpages/isomercms-frontend/commit/8484744fba0d59973275b3341d90f8e840221038